src/url.js

```diff
@@ -156,68 +289,38 @@ function finalizeResourceType(resourceType = "image", type = "upload", urlSuffix
  * @param {type} [options.type='upload'] - The asset's storage type.
  *  For details on all fetch types, see
  * <a href="https://cloudinary.com/documentation/image_transformations#fetching_images_from_remote_locations"
- *  target="_blank">Fetch types</a>. 
- * @param {Object} [options.resource_type='image'] - The type of asset. <p>Possible values:<br/> 
+ *  target="_blank">Fetch types</a>.
+ * @param {Object} [options.resource_type='image'] - The type of asset. <p>Possible values:<br/>
  *  - `image`<br/>
  *  - `video`<br/>
- *  - `raw` 
+ *  - `raw`
```

Comment:
@maoznir, please update the doc string

Reply:
in what way?

Reply:
done

Reply:
@maoznir , is type and resource_type sent separately to this method?